apps/requestron/lib/needsResponse.js

```diff
@@ -1,54 +1,90 @@
         // get a list of open issues in the repo
         const issue_response = await instance.get('repos/' + process.env.REPO_OWNER + '/' + process.env.REPO_NAME + '/issues');
         const issues = issue_response["data"];
 
         // get a list of team members
-        const team_response = await instance.get('/orgs/' + process.env.OPS_TEAM_ORG + '/teams/' + process.env.OPS_TEAM_NAME + '/members');
-        const team = team_response["data"];
-        let team_members = [];
-        for (let i in team) {team_members.push(team[i].login)}
+        // yes, i know hardcoding it is bad.
+        // i gotta come up with a better option, honestly.
+        let team_members = ["caggles", "ShellyXueHan", "patricksimonian"];
 
         //for each open issue, check the most recent comment
         for (let i in issues) {
 
             let comment_response = await instance.get('repos/' + process.env.REPO_OWNER + '/' + process.env.REPO_NAME + '/issues/' + issues[i].number + '/comments');
-            let comments = comment_response["data"];
+            let unfiltered_comments = comment_response["data"];
+
+            //remove bot comments from the list of comments because we don't want them to change anything
+            let comments = [];
+            for (let k in unfiltered_comments) {
+                if (unfiltered_comments[k].user.login !== process.env.APP_NAME + '[bot]') {
+                    comments.push(unfiltered_comments[k])
+                }
+            }
 
             //make a nice little list of the current labels
             let current_labels=[];
             for (let j in issues[i].labels) {current_labels.push(issues[i].labels[j].name)}
 
             //get  the most recent comment
-            let recent_comment = comments[issues[i].comments - 1];
+            let recent_comment = undefined;
+            if (comments.length > 0) {
+                recent_comment = comments[comments.length - 1];
+            }
+
 
             if (recent_comment !== undefined) {
 
-                //if the most recent comment is from an ops team member (or is excluded from staleness tracking) and the client-response label is on the ticket, remove the client-response label
+                //if the most recent comment is from an ops team member (or is excluded from staleness tracking) and the client-response label is on the ticket.
                 if ((team_members.includes(recent_comment.user.login) || current_labels.includes('staleness-exception')) &&
                     current_labels.includes('client-response')) {
-                    await instance.delete('/repos/' + process.env.REPO_OWNER + '/' + process.env.REPO_NAME + '/issues/' + issues[i].number + '/labels/client-response')
 
+                    await instance.delete('/repos/' + process.env.REPO_OWNER + '/' + process.env.REPO_NAME + '/issues/' + issues[i].number + '/labels/client-response');
+                    console.log('client-response removed on issue #' + issues[i].number);
```

Comment:
you probably want to consider output logs to a file as console logs go away for sometime